software_subprojects.md

```diff
@@ -0,0 +1,31 @@
+# Software Subprojects of Project Jupyter
+
+Software design and development in Project Jupyter is organized into a set of Software Subprojects. These software Subprojects often map onto GitHub repositories/organizations, but that is not strictly required. The lifecycle of Software Subprojects is [described in detail here](newsubprojects.md).
+
+Since Project Jupyter promotes open technical standards and interoperability among components, there is a rich ecosystem of tools developed by third parties that are not under the umbrella of Project Jupyter’s Governance or the Software Steering Council. We strive to maintain open communication with this community, however,this document focuses only on official Jupyter Subprojects.
+
+## Responsibilities of Jupyter Subprojects
+
+Unless the SSC or the Board of Directors says otherwise, Subprojects self-govern as autonomously as possible, while following the overall governance model and processes of Project Jupyter. Specifically, all Subprojects under Jupyter’s governance have the following responsibilities:
+
+- Adhere to the Jupyter Code of Conduct.
+- Adhere to the [Jupyter Decision Making Guidelines and process](decision_making.md).
+- Where applicable, nominate and maintain a single representative to the SSC.
+- Follow the licensing guidelines and practices of the project ([use of the BSD license and copyright header](https://github.com/jupyter/jupyter/blob/master/LICENSE)).
+- Follow Jupyter’s trademark, branding, and intellectual property guidelines.
+- Conduct its activities in a manner that is open, transparent, and inclusive. This includes coordinating with the SSC and the Board of Directors on mechanisms for information flow and updates to the broader community (details of this, project-wide, will be developed as our new governance model is adopted and implemented).
```

Comment:
Given the use of Team Compass in several orgs, this should be required of each subproject. It will satisfy open and transparent and make it structured vs ad hoc by each group.

Reply:
Fully agreed, excellent idea.

Reply:
BTW - we made it an explicit requirement to make sure  it didn't fall through the cracks.

Furthermore - over time those can evolve a bit of machine-readable metadata that could be mined across the project to help maintain common stats, an overall view, etc.

Reply:
Is this something that could be stewarded by the standards subgroup? I think “defining standards for sub project policies and practices” would be helpful and also require expertise to get right, and may evolve over time. That feels like work for an ongoing group to me